scripts/install.sh

```diff
@@ -116,7 +133,7 @@ fi
 
 
 # Create a symlink
-if ! sudo ln -sf "$FVM_DIR_BIN/fvm" "$SYMLINK_TARGET"; then
+if ! "$ESCALATION_TOOL" ln -sf "$FVM_DIR_BIN/fvm" "$SYMLINK_TARGET"; then
```

Comment:
For me on Ubuntu 24.04.1 LTS in a Konsole shell, the order of these arguments had to be swapped to get this install script to stop reporting `"Failed to create symlink."`

Reply:
I had to manually run `sudo ln -s /home/$(whami)/.fvm_flutter/bin/fvm /usr/local/bin/fvm` to get it working.

Reply:
The order is correct here mate. The fvmdirbin is /home/user/.fvm_flutter/bin and the symlink target is /usr/local/bin/fvm

Reply:
That's so strange, because it didn't work for me unless I swapped the argument order. 
https://en.wikipedia.org/wiki/Ln_(Unix)#Specification

Reply:
First it should be the `source` then the `destination`. Or do you mean the `-sf` arguments order?

Reply:
It should work regardless of the order of `-sf`

Reply:
After double checking, you're right. 
The issue I was experiencing must've been from an env var not being set higher up in the script. 
I did still have to manually create the symlink; but this isn't the culprit.